src/api/entityResolution/index.ts

```diff
@@ -273,4 +274,25 @@ export class EntityResolutionAPI extends Request {
       params: params
     });
   }
+
+  /**
+   * Get entity resolution server status
+   */
+  getEntityResolutionServerStatus(
+    this: Request<{status: string}>,
+    params: GetEntityResolutionServerStatusParams
+  ) {
+    return this.request({
+      errors: [
+        UNAUTHORIZED,
+        FORBIDDEN,
+        DATA_SOURCE_UNAVAILABLE,
+        NOT_FOUND,
+        ENTITY_RESOLUTION_EXPIRED_LICENSE
```

Comment:
We also need to add the error `FEATURE_DISABLED`  in case the API is called and ER is not enabled
see:
https://github.com/Linkurious/linkurious-server/blob/faca79f88479f4763b51d36e7240ed06898cf068/server/routes/entityResolution.ts#L47-L48

Reply:
I don't see FEATURE_DISABLED in any of the routes, I am not sure if we should add it only here